src/client/css/partials/unsubscribe.css

```diff
@@ -0,0 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.unsubscribe {
```

Comment:
There is currently no specific design in the current version of Monitor and also no spec for the refresh. I’ll sync with UX if there is anything they’d like to add.